packaging/installer/kickstart.sh

```diff
@@ -1630,38 +1683,8 @@ while [ -n "${1}" ]; do
   shift 1
 done
 
-check_claim_opts
-confirm_root_support
-get_system_info
-confirm_install_prefix
-
-if [ "${ACTION}" = "uninstall" ]; then
-  uninstall
-  cleanup
-  trap - EXIT
-  exit 0
-fi
-
-tmpdir="$(create_tmp_directory)"
-progress "Using ${tmpdir} as a temporary directory."
-cd "${tmpdir}" || fatal "Failed to change current working directory to ${tmpdir}." F000A
-
-handle_existing_install
-
-case "${SYSTYPE}" in
-  Linux) install_on_linux ;;
-  Darwin) install_on_macos ;;
-  FreeBSD) install_on_freebsd ;;
-esac
-
-if [ -n "${NETDATA_CLAIM_TOKEN}" ]; then
-  claim
-elif [ "${NETDATA_DISABLE_CLOUD}" -eq 1 ]; then
-  soft_disable_cloud
+if [ -z "${ACTION}" ]; then
+  handle_existing_install
```

Comment:
This needs `check_claim_opts`, `confirm_root_support`, `get_system_info`, and `confirm_install_prefix` run before it is run, otherwise it will not work reliably.